src/api-doc/api-doc.yaml

```diff
@@ -4436,92 +4439,205 @@ paths:
-                status:
-                  type: string
-                  description: Status of the comment (e.g., "OPEN" / "DRAFT" , "RESOLVED").
-                parent_id:
-                  type: string
-                  description: ID of the parent comment, if applicable
-
+                  oneOf:
+                    - type: object
+                      properties:
+                        id:
+                          type: integer
+                          description: ID of the comment
+                        resource_id:
+                          type: integer
+                          description: ID of the resource associated with the comment
+                        comment:
+                          type: string
+                          description: The actual content of the comment
+                        user_id:
+                          type: string
+                          description: ID of the user who made the comment
+                        parent_id:
+                          type: integer
+                          description: ID of the parent comment, if applicable
+                        status:
+                          type: string
+                          description: Status of the comment (e.g., "OPEN", "DRAFT", "RESOLVED")
+                        context:
+                          type: string
+                          description: Context of the comment (e.g., "page" or any tag)
+                        page:
+                          type: string
+                          description: Page number associated with the comment (e.g., "1")
+                        is_read:
+                          type: boolean
+                          description: Whether the comment has been read
+                    - type: array
+                      items:
+                        type: object
+                        properties:
+                          id:
+                            type: integer
+                            description: ID of the comment
+                          resource_id:
+                            type: integer
+                            description: ID of the resource associated with the comment
+                          comment:
+                            type: string
+                            description: The actual content of the comment
+                          user_id:
+                            type: string
+                            description: ID of the user who made the comment
+                          parent_id:
+                            type: integer
+                            description: ID of the parent comment, if applicable
+                          status:
+                            type: string
+                            description: Status of the comment (e.g., "OPEN", "DRAFT", "RESOLVED")
+                          resolved_by:
+                            type: string
```

Comment:
@adithyadinesh0412  as discussed please remove these keys from req body

Reply:
this was asked by QA team to add applicable statuses in the doc @priyanka-TL